site/assets/scripts/chart.js

```diff
@@ -0,0 +1,88 @@
+var chartInstance = null;
+
+function createChart (sessionsByDay, month) {
+    var hoverLabels = [];
+    var displaySessions = [];
+
+    for (const [key, value] of Object.entries(sessionsByDay)) {
+        var index = parseInt(key);
+        hoverLabels[index - 1] = month + " " + index;
+        displaySessions[index - 1] = value;
+    }
+
+    const ctx = document.getElementById('activityChart');
+
+    chartInstance = new Chart(ctx, {
+        type: 'bar',
+        data: {
+            labels: hoverLabels,
+            datasets: [{
+                data: displaySessions,
+                borderWidth: 1,
+                backgroundColor: '#3E3498',  // This is the secondary theme color in _theme.scss
+            }]
+        },
+        options: {
+            scales: {
+                x: {
+                    display: false
+                },
+                y: {
+                    beginAtZero: true,
+                    ticks: {
+                        callback: function(value, index, ticks) {
+                            if (value >= 1000) {
+                                return (value / 1000) + 'k';
+                            }
+                            else {
+                                return value;
+                            }
+                        }
+                    }
+                },
+            },
+            responsive: true,
+            maintainAspectRatio: false,
+            plugins: {
+                legend: {
+                display: false
+                },
+                tooltip: {
+                    xAlign: 'center',
+                    yAlign: 'bottom',
+                    backgroundColor: '#FFFFFF',
+                    bodyColor: '#000000',
+                    titleColor: '#000000',
+                    borderColor: '#3E3498',  // This is the secondary theme color in _theme.scss
+                    borderWidth: 2,
+                    displayColors: false,
+                    titleFont: {weight: 'normal'},
+                    bodyFont: {weight: 'bold'},
+                    callbacks: {
+                        label: function(context) {
+                            let label = context.parsed.y + ' sessions';
```

Comment:
The number of sessions in the hover label needs to be K'd and rounded to two decimal places to match the mockup

![kilo](https://user-images.githubusercontent.com/46821257/224085533-1836a52e-0926-4a6d-87e6-7037a3634a9d.png)